### DISCUSSION TITLE
How to deploy using small size RDS and EC2?

### DISCUSSION THREAD

**Discussion by blueandhack** [magma/magma#8645] [Q&A]
I tried t3.small (EC2) and m3.large (RDS), but not lucky.
So, what is the minimum size, for instance? Or what is the required minimum size? Thanks.

**Comment by joary** [comment]
Curious, what kind of issues did you found during the deployment?

**Reply by blueandhack** [reply]
We are just during testing, so we want to reduce costs.
Currently, we have just had one AGW and three UEs.
